src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * Function takes a string with styles
+ * (see an example in
+ * [stylesString.js](./stylesString.js))
+ * and returns an object where CSS properties
+ * are keys
+ * and values are the values of related CSS
+ * properties (see an exampl
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const obj = {};
+  const arrWithStyles = sourceString.split(';');
```

Comment:
It is better to name variables of these types - nameArr (in your case - stylesArr), this is considered a good practice.